src/apps/code-editor/src/app/components/FileDrawer/FileDrawer.js

```diff
@@ -82,9 +85,12 @@ export const FileDrawer = memo(function FileDrawer(props) {
 
   function handleSetOpen() {
     // TODO persist to user settings
-    setOpen(!open);
+    setOpen((open) => !open);
   }
 
+  // Handle keyboard shortcut Drawer
+  useKeyboardShortcut("d", () => handleSetOpen());
```

Comment:
The callback does not need to be wrapped in a function. It should be passed directly.

Reply:
I thought this was best practice to prevent bugs for asynchronous code, if I remove the callback then my code doesn't work.